src/NosCore.Parser/Parsers/ItemParser.cs

```diff
@@ -0,0 +1,679 @@
+                {nameof(ItemDto.Element), chunk => ImportElement(chunk)},
+                {nameof(ItemDto.MaxCellonLvl), chunk => ImportMaxCellonLvl(chunk)},
+                {nameof(ItemDto.MaxCellon), chunk => ImportMaxCellon(chunk)},
+                {nameof(ItemDto.DistanceDefenceDodge), chunk => ImportDistanceDefenceDodge(chunk)},
+                {nameof(ItemDto.MaximumAmmo), chunk => ImportMaximumAmmo(chunk)},
+                {nameof(ItemDto.CriticalRate), chunk => ImportCriticalRate(chunk)},
+                {nameof(ItemDto.CriticalLuckRate), chunk => ImportCriticalLuckRate(chunk)},
+                {nameof(ItemDto.HitRate), chunk => ImportHitRate(chunk)},
+                {nameof(ItemDto.DamageMaximum), chunk => ImportDamageMaximum(chunk)},
+                {nameof(ItemDto.DamageMinimum), chunk => ImportDamageMinimum(chunk)},
+            };
+
+            var genericParser = new GenericParser<ItemDto>(folder + ItemCardDto,
+                "#========================================================", 1, actionList, _logger);
+            var items = genericParser.GetDtos().GroupBy(p => p.VNum).Select(g => g.First()).ToList();
+            foreach (var item in items)
+            {
+                HardcodeItem(item);
+                if (item.ItemType == ItemType.Specialist)
+                {
+                    var elementdic = new Dictionary<ElementType, int> {
+                        { ElementType.Neutral, 0 },
+                        { ElementType.Fire, item.FireResistance },
+                        { ElementType.Water, item.WaterResistance },
+                        { ElementType.Light, item.LightResistance },
+                        { ElementType.Dark, item.DarkResistance }
+                    }.OrderByDescending(s => s.Value).ToList();
+
+                    item.Element = elementdic.First().Key;
+                    if (elementdic.First().Value != 0 && elementdic.First().Value == elementdic.ElementAt(1).Value)
+                    {
+                        item.SecondaryElement = elementdic.ElementAt(1).Key;
+                    }
+                }
+            }
+            SetVehicles(items, new Dictionary<byte, List<(short, short)>>
+            {
+                { 20, new List<(short, short)> { (9054, 2368), (1906, 2368), (9055, 2370), (1907, 2370), (9058, 2406), (1965, 2406), (9065, 2411),
+                    (5008, 2411), (5238, 1817), (5226, 1817), (5240, 1819), (5228, 1819), (5234, 2522), (5236, 2524) } },
+                { 21, new List<(short, short)> { (9070, 2429), (5117, 2429), (9073, 2432), (5152, 2432), (5196, 2517), (9078, 2520), (5232, 2520),
+                    (5321, 2528), (9090, 2934), (5386, 2934), (9091, 2936), (5387, 2936), (9092, 2938), (5388, 2938), (9093, 2940), (5389, 2940),
+                    (9094, 2942), (5390, 2942), (5391, 2944), (9115, 3679), (5997, 3679), (9079, 2522), (9080, 2524), (9081, 1817), (9082, 1819) } },
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Move this closing curly brace to the next line.](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4456396)